From 7e7e970e02a0be5902bdccd62a7945189328fbc4 Mon Sep 17 00:00:00 2001
From: Miroslaw Swiader <miroslaw.swiader@intel.com>
Date: Mon, 18 Apr 2016 17:40:41 +0200
Subject: [PATCH] xhci: KlockWork Fixes

Pointer will be dereferenced after it was positively
checked for NULL

Change-Id: I70cacb096b1921896905fc6a0be3d1e887a47343
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-29527
Signed-off-by: Miroslaw Swiader <miroslaw.swiader@intel.com>
Reviewed-on: https://android.intel.com:443/495898
---
 drivers/usb/host/xhci-intel-cap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/xhci-intel-cap.c b/drivers/usb/host/xhci-intel-cap.c
index fbc7d0a..ac26fb8 100644
--- a/drivers/usb/host/xhci-intel-cap.c
+++ b/drivers/usb/host/xhci-intel-cap.c
@@ -303,7 +303,7 @@ void xhci_intel_pipe_sync_phystatus_quirk(struct xhci_hcd *xhci, bool enable)
 	u32	data;
 
 	if (!xhci || !xhci->phy_mux_regs) {
-		xhci_err(xhci, "no xhci or phy_mux_regs\n");
+		pr_err("no xhci or phy_mux_regs\n");
 		return;
 	}
 
-- 
1.9.1

